arch/X86/X86GenAsmWriter1.inc

```diff
@@ -9,6 +9,8 @@
 /* Capstone Disassembly Engine, http://www.capstone-engine.org */
 /* By Nguyen Anh Quynh <aquynh@gmail.com>, 2013-2015 */
 
+#ifndef CAPSTONE_DIET
```

Comment:
yes, for example printInstruction() calls printOperand(): https://github.com/aquynh/capstone/blob/master/arch/X86/X86IntelInstPrinter.c#L583